daprdocs/content/en/developing-applications/building-blocks/state-management/query-state-store-api.md

```diff
@@ -0,0 +1,265 @@
+---
+type: docs
+title: "State Query API"
+linkTitle: "State Query API"
+weight: 600
+description: "API for querying state stores"
+---
+
+## Introduction
+
+The query API provides unified way of querying state stores regardless of underlying databases.
```

Comment:
Provide a more descriptive intro. 

The query API provides a way of querying the key/value data stored in state store components. This query API is not a replacement for a complete query language, and is focused on retrieving, filtering and sorting key/value data that you have saved through the state management APIs. 

This query API does not support querying of actor state stored in a state store. For that you need to use the query API for the specific database.  See [query actor state)({{< ref state-management-overview/#querying-actor-state >}})